### PR TITLE
perplexity: update README FP16 results [no ci]

### PR DESCRIPTION
**Pull request by JohannesGaessler**
The logits used for comparative runs of `perplexity` are stored as `uint16_t` instead of `float`. The difference from this downcasting can be non-negligible when looking at quants like q8_0 or q6_K. This PR adds a disclaimer and results to estimate the impact of the downcasting.

### CODE REVIEW

**Review by slaren — APPROVED** [review]

**Comment by fedric95** [comment]
@JohannesGaessler great work, plans to add also the scoreboard for llama3-70b?

**Comment by JohannesGaessler** [comment]
I'm hesitant to publish anything with LLaMA 3 70b because it turned out that the machine I built with 6x RTX 4090 has stability issues which means I have to be very careful that the data isn't being affected by random bit flips.

**Comment by fedric95** [comment]
> I'm hesitant to publish anything with LLaMA 3 70b because it turned out that the machine I built with 6x RTX 4090 has stability issues which means I have to be very careful that the data isn't being affected by random bit flips.

:-( how much time does it take to run all the experiments for llama3 70b (more or less)? Just to understand how much it would cost.

**Comment by JohannesGaessler** [comment]
At standard settings a single LLaMA 3 70b run takes ~6 minutes on 6x RTX 4090.
